apollo-core/src/main/java/com/ctrip/framework/foundation/internals/NullProviderManager.java

```diff
@@ -32,6 +34,11 @@ public NullProvider provider(Class clazz) {
     return provider;
   }
 
+  @Override
+  public int getOrder() {
```

Comment:
as ProviderManager has a default getOrder implementation, I think we could omit this here?